cwf/cloud/go/go.mod

```diff
@@ -11,7 +11,9 @@
 //
 module magma/cwf/cloud/go
 
+// TODO remove golang.org/x/net line once Go Upgrade is merged
```

Comment:
Minor comment: It would be best practice to actually link the PR, that is doing the go upgrade